code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -110,6 +110,10 @@
 	start_process()
 
 
+/obj/structure/cult/pylon/Destroy()
+	processing_objects.Remove(src)

```

Comment:
I dont see a need to have any of those in destroy. Any references held in them will cease to exist when the pylon is gone anyways.

I remove them from processing objects because thats the opposite situation. Its a reference **something else** is holding **about the pylon** and thus interferes with it being deleted. The five things listed above are references the pylon holds about other thing